content/en/apps/guides/security/securing-android.md

```diff
@@ -46,11 +46,7 @@ Android does not support disk encryption in versions earlier than 3.0
 
 ## Mobile device management
 
-Setting up mobile device management that if a phone gets lost or stolen you can ensure the data on it is deleted safely.
+Using mobile device management (MDM) software allows administrators to remotely manage mobile devices, which often includes the option to delete apps and data from lost or stolen mobile devices.
```

Comment:
@abbyad This feels a little duplicated with the documentation here: https://docs.communityhealthtoolkit.org/core/guides/android/publishing/#mobile-device-management

What do you think about updating the "publishing" page with the additional information you've added here, and then linking from here to the "publishing" page?

Reply:
Opened #717 with corresponding changes.